### ISSUE TITLE
Consider adding functional tests to this project

### ISSUE THREAD

**Issue by danielbachhuber** [iandunn/wp-cli-rename-db-prefix#1]
Functional tests are an integral ingredient of highly-quality, maintainable commands. WP-CLI tries to make it as easy as possible to add functional tests to your package with its `wp scaffold package-tests` command:

https://github.com/wp-cli/scaffold-package-command#wp-scaffold-package-tests

I'd encourage you to consider adding functional tests to your package :) By starting your functional tests early on, it also makes it much easier to maintain your project over time.

**Comment by iandunn** [comment]
Yup, that would be a great improvment. I haven't tried out Behat yet either, so it'd be a good excuse to do that as well. Thanks :)

**Comment by szepeviktor** [comment]
It is very easy for wp-cli because all functions are ready you only have to write plain English.

**Comment by szepeviktor** [comment]
Moving code comments here

- can be in tests dir intead of features?
- prune unused stuff that `scaffold package-tests` added
